scripts/regenerate.sh

```diff
@@ -38,39 +38,42 @@ echo "go install cmd/protoc-gen-go-grpc"
 (cd cmd/protoc-gen-go-grpc && go install .)
 
 echo "git clone https://github.com/grpc/grpc-proto"
-git clone --quiet https://github.com/grpc/grpc-proto ${WORKDIR}/grpc-proto
+git clone --quiet https://github.com/grpc/grpc-proto "${WORKDIR}/grpc-proto"
 
 echo "git clone https://github.com/protocolbuffers/protobuf"
-git clone --quiet https://github.com/protocolbuffers/protobuf ${WORKDIR}/protobuf
+git clone --quiet https://github.com/protocolbuffers/protobuf "${WORKDIR}/protobuf"
 
 # Pull in code.proto as a proto dependency
-mkdir -p ${WORKDIR}/googleapis/google/rpc
+mkdir -p "${WORKDIR}/googleapis/google/rpc"
 echo "curl https://raw.githubusercontent.com/googleapis/googleapis/master/google/rpc/code.proto"
-curl --silent https://raw.githubusercontent.com/googleapis/googleapis/master/google/rpc/code.proto > ${WORKDIR}/googleapis/google/rpc/code.proto
+curl --silent https://raw.githubusercontent.com/googleapis/googleapis/master/google/rpc/code.proto > "${WORKDIR}/googleapis/google/rpc/code.proto"
 
-mkdir -p ${WORKDIR}/out
+source ./scripts/protoc_installer.sh
+install_protoc $WORKDIR
```

Comment:
I don't believe we should do this always.  On local runs, I don't want every `regenerate` run to download a binary.

Maybe if `protoc` is installed and in your path, we can run `protoc --version`, and if it prints what we want, we proceed, otherwise print a warning that the version is wrong and you should think about running `protoc_installer.sh` to fix it.

Reply:
How about we check for the desired protoc version and install it in case we don't have a match?

Reply:
I'd rather not install anything on local runs unless the user explicitly performs an action to do so.

Reply:
updated